lib/poxa/registry.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Poxa.Registry do
+  @callback register!(String.t) :: any
+  @callback register!(String.t, any) :: any
+
+  @callback unregister!(String.t) :: any
+
+  @callback send!(String.t, String.t, PresenceSubscription.user_id | pid) :: any
+  @callback send!(String.t, String.t, any, String.t) :: any
+
+  @callback subscription_count(String.t, PresenceSubscription.user_id | pid | :_) :: non_neg_integer
+
+  @callback subscriptions(pid | :_) :: Map.t
+
+  @doc """
```

Comment:
I just copied from somewhere else when I was moving the code here :see_no_evil:
Don't worry, it's in my TODO list for this PR to write the docs for these callbacks :wink: